metricbeat/magefile.go

```diff
@@ -133,11 +127,35 @@ func MockedTests(ctx context.Context) error {
 	return devtools.GoTest(ctx, params)
 }
 
-// Fields generates a fields.yml for the Beat.
-func Fields() error {
+// Fields generates a fields.yml and fields.go for each module.
+func Fields() {
+	mg.Deps(fieldsYML, moduleFieldsGo)
+}
+
+func fieldsYML() error {
 	return devtools.GenerateFieldsYAML("module")
 }
 
+func moduleFieldsGo() error {
+	return devtools.GenerateModuleFieldsGo("module")
+}
+
+// Update is an alias for running fields, dashboards, config.
+func Update() {
+	mg.SerialDeps(Fields, Dashboards, Config,
+		metricbeat.PrepareModulePackagingOSS,
+		metricbeat.GenerateOSSMetricbeatModuleIncludeListGo)
+}
+
+// ExportDashboard exports a dashboard and writes it into the correct directory
+//
+// Required ENV variables:
+// * MODULE: Name of the module
+// * ID: Dashboard id
+func ExportDashboard() error {
```

Comment:
There should be a `dashboards:export` target already. So this shouldn't be needed.

Reply:
Removed, and verified that `dashboards:export` is there.